vagrant/Vagrantfile

```diff
@@ -73,7 +73,7 @@ Vagrant.configure("2") do |config|
        "--type", "dvddrive",
        "--medium", "emptydrive"
        ]
-	vb.name = "SIRF_2.0.0-rc.1"
+	vb.name = "SIRF_2.0.0-rc.3"
```

Comment:
I'd already put it 2.0.0. That way we can simply retag if it all works. (same everywhere)